tests/snippets/test_frame.py

```diff
@@ -500,3 +500,37 @@ def test_types_from_dict() -> None:
     pd.DataFrame.from_dict({'a': {'row1': 2}, 'b': {'row2': 4, 'row1': 4}})
     pd.DataFrame.from_dict({'a': (1, 2, 3), 'b': (2, 4, 5)})
     pd.DataFrame.from_dict(data={'col_1': {'a': 1}, 'col_2': {'a': 1, 'b': 2}}, orient="columns")
+
+
+def test_pipe() -> None:
+    def foo(df: pd.DataFrame) -> pd.DataFrame:
+        return df
+
+    df1: pd.DataFrame = (
+        pd.DataFrame({'a': [1]})
+        .pipe(foo)
+    )
+
+    df2: pd.DataFrame = (
```

Comment:
These extra brackets are bothering me a little. It could be just my personal taste though so I am approving anyway.

Reply:
Thanks! I've addressed this in a comment below.